emission/simulation/trip_gen.py

```diff
@@ -25,30 +27,33 @@ class Creator:
     def __init__(self):
         self.starting_points = [ ]
         self.ending_points = [ ]
-        self.a_to_b = []
+        self.a_to_b = [ ]
         self.num_trips = None
         self.radius = None
         self.amount_missed = 0
-        self.starting_addresses = []
-        self.ending_addresses = []
-        self.labels = []
+        self.starting_addresses = [ ]
+        self.ending_addresses = [ ]
+        self.labels = [ ]
 
     def get_starting_ending_points(self):
-        city_file = open("CFC_DataCollector/trip_generator/input.json", "r") ## User (Naomi) specifies locations and radius they want
+        city_file = open("emission/simulation/input.json", "r") ## User (Naomi) specifies locations and radius they want
```

Comment:
I agree. We were thinking of doing that but we decided to leave it this way for now.